mono/mini/interp/interp.c

```diff
@@ -229,7 +229,7 @@ int mono_interp_traceopt = 0;
 #define MINT_IN_DEFAULT default:
 #endif
 
-static void
+static MONO_NEVER_INLINE void // Inlining this causes caller to use more stack.
 set_resume_state (ThreadContext *context, InterpFrame *frame)
```

Comment:
The comment is kinda not neccessary, we do it for the same reason for all functions.

Reply:
It isn't obvious here since there are no locals.
I guess it increases register pressure in caller.

Reply:
But ok either way, I can remove it.